src/main/java/org/braekpo1nt/mctmanager/games/mecha/MechaGame.java

```diff
@@ -335,7 +343,23 @@ public void onPlayerDeath(PlayerDeathEvent event) {
             onTeamWin(winningTeam);
         }
     }
-    
+
+    @EventHandler // prevent player interaction with Item Frames or Armor Stands
+    public void onPlayerInteract(PlayerInteractEvent event) {
+        if (!gameActive) {
+            return;
+        }
+        if (!mechaHasStarted) {
+            return;
+        }
+        if (event.getClickedBlock() != null) {
+            Material blockType = event.getClickedBlock().getType();
+            if (blockType == Material.ARMOR_STAND || blockType == Material.ITEM_FRAME) {
```

Comment:
There's no way you would know this until I told you, but armor stands and item frames are entities, not blocks.